infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     {{- with .Values.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
+        # instrumentation.opentelemetry.io/inject-python: "true"
```

Comment:
does this need to stay? a check added?

Reply:
for sure! i was thinking it made more sense though to incorporate it into the metrics param chart checks. e.g. 
```yaml
{{- if .Values.metrics }}
```